src/docs/cookbook/animation/opacity-animation.md

```diff
@@ -10,7 +10,8 @@ quickly popping elements on and off the screen can feel jarring to end users.
 Instead, we can fade elements in and out with an opacity animation to create
 a smooth experience.
 
-In Flutter, we can achieve this task using the [`AnimatedOpacity`](https://docs.flutter.io/flutter/widgets/AnimatedOpacity-class.html)
+In Flutter, we can achieve this task using the
```

Comment:
Not crucial for *this* URL, but in general it'd be nice to put the link definitions after the text, rather than inline (as per https://github.com/dart-lang/site-shared/blob/master/doc/markdown.md#prefer-placing-link-definitions-after-the-text-containing-them).